driver.sh

```diff
@@ -292,7 +304,14 @@ build_linux() {
 }
 
 boot_qemu() {
-  local kernel_image=${tree}/arch/${ARCH}/boot/${image_name}
+  local kernel_image
+
+  if [[ ${image_name} = "vmlinux" ]]; then
```

Comment:
What's up with `image_name`? Does the mips build not place the resulting image in `arch/mips/boot/vmlinux`?

Reply:
No. `vmlinux` is just the raw kernel image, which always ends up in the root of the kernel or object directory. `malta_defconfig` appears to generate a binary kernel image `vmlinux.bin` in `arch/mips/boot/compressed` but it seems like most people just use the regular kernel image.

Reply:
Are there no kernel binary images in `arch/mips/boot/` (not `compressed`)?

Reply:
```
$ fd --no-ignore --maxdepth 1 . out/arch/mips/boot
out/arch/mips/boot/compressed
out/arch/mips/boot/dts
out/arch/mips/boot/tools
```

Reply:
what if you set:

```
image_name = "compressed/vmlinux.bin.z"
```
in the mips block, then you don't need this???

Reply:
I'll see if that works.

Reply:
```
+ timeout 2m unbuffer qemu-system-mipsel -m 512m -machine malta -cpu 24Kf -append root=/dev/sda -drive file=images/mipsel/rootfs.ext4,format=raw,if=ide -display none -serial mon:stdio -kernel linux/arch/mips/boot/vmlinux.bin
qemu-system-mipsel: could not load kernel 'linux/arch/mips/boot/vmlinux.bin': The image is not ELF
```

Reply:
yep...strange inconsistency, but w/e